storm-core/src/clj/org/apache/storm/daemon/executor.clj

```diff
@@ -265,6 +265,7 @@
      :stats (mk-executor-stats <> (sampling-rate storm-conf))
      :interval->task->metric-registry (HashMap.)
      :task->component (:task->component worker)
+     :task-metrics (TaskMetrics/taskMetricsMap (first task-ids) (last task-ids) worker-context component-id)
```

Comment:
We can create TaskMetrics instance in task-data and even get rid of the map lookup. I guess TaskMetrics doesn't have any executor specific metrics.

Reply:
Never mind. Just fixed and pushed new commit.

Reply:
@HeartSaVioR Nice, thanks!